src/components/Calendar/Calendar.css

```diff
@@ -276,6 +276,11 @@ button.rbc-input::-moz-focus-inner {
       -moz-transition: all .3s ease-in-out;
       -webkit-transition: all .3s ease-in-out;
     }
+
+    .rbc-date-cell.rbc-now > a:focus, .rbc-date-cell.rbc-now.rbc-current > a:focus {
```

Comment:
Add each class declaration on it's own line and remove nesting indents.

Reply:
i removed nesting declaration

Reply:
@Farhan-emgage I see that the declaration are now on their own lines, however the code is still nested. Can you please remove the nesting. Thanks

Reply:
as your comment i understand that you are asking about i declare css property like ".rbc-date-cell.rbc-now > a:focus". and you want to use it like 
.rbc-date-cell{
     .rbc-now{
          a{
            &:focus{
             }
          }
     }
}

Reply:
@Farhan-emgage by nesting I mean:

.classname {
  styles...
}
    .classname {
      styles...
    }
        .classname {
          styles...
        }

Which is very confusing, as it makes it look like the styles are actually nested as is done is scss, like so:

.classname {
  styles...

    .classname {
      styles...

        .classname {
          styles...
        }
    }
}

If they are not actually nested styles, then they should not appear nested, but should have a flat structure like so:

.classname {
  styles...
}
.classname {
  styles...
}
.classname {
  styles...
}

Reply:
@Farhan-emgage can you please take care of the above request so we can get this merged.